packages/react-dom/src/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -18,11 +18,15 @@ class ReactMarkupReadableStream extends Readable {
     this.partialRenderer = new ReactPartialRenderer(element, makeStaticMarkup);
   }
 
+  _destroy() {
+    this.partialRenderer.destroy();
+  }
+
   _read(size) {
     try {
       this.push(this.partialRenderer.read(size));
     } catch (err) {
-      this.emit('error', err);
```

Comment:
is this breaking?

Reply:
what is it fixing?

Reply:
I submitted a PR to fix this: https://github.com/facebook/react/pull/14314